component/portal/src/test/resources/org/exoplatform/portal/config/conf/portal/classic/portal.xml

```diff
@@ -83,5 +83,6 @@
       <show-info-bar>false</show-info-bar>
     </portlet-application>
   </portal-layout>
-
+  <displayed>true</displayed>
+  <display-order>1</display-order>
```

Comment:
the properties definition should go first in XML definition (XSD), and after, the XML child nodes. This strategy mixes and defines no order in XSD IMO which makes the XML hard to read.

Reply:
done